libse/Forms/DurationsBridgeGaps.cs

```diff
@@ -19,7 +19,12 @@ public static int BridgeGaps(Subtitle subtitle, int minMsBetweenLines, bool divi
             for (int i = 0; i < count; i++)
             {
                 var cur = subtitle.Paragraphs[i];
-                var next = subtitle.Paragraphs[i + 1];
```

Comment:
this is very likely to throw exception when subtitle contains only one paragraph